sdk/nodejs/entrypoint/register.ts

```diff
@@ -119,6 +119,8 @@ function addTypeDef(type: ScannerTypeDef<TypeDefKind>): TypeDef {
       return dag.typeDef().withObject((type as ObjectTypeDef).name)
     case TypeDefKind.Listkind:
       return dag.typeDef().withListOf(addTypeDef((type as ListTypeDef).typeDef))
+    case TypeDefKind.Voidkind:
+      return dag.typeDef().withKind(type.kind).withOptional(true)
```

Comment:
I don't see why _optional_ is enforced here.
```suggestion
```

Reply:
Yeah, true! I think the server should enforce it though (i.e., setting optional to true on `withKind("void")`).